src/Mod/Draft/draftguitools/gui_snapper.py

```diff
@@ -1007,20 +1072,23 @@ def snapToSpecials(self, obj, lastpoint=None, eline=None):
 
         return snaps
 
+
     def getScreenDist(self, dist, cursor):
         """Return a distance in 3D space from a screen pixels distance."""
         view = Draft.get3DView()
         p1 = view.getPoint(cursor)
         p2 = view.getPoint((cursor[0] + dist, cursor[1]))
         return (p2.sub(p1)).Length
 
+
```

Comment:
These methods are inside a class, so they only need one newline of space between them. For methods outside of a class, that is, functions, the spacing is two empty newlines.